OPUNetTransportLayer.cpp

```diff
@@ -1101,6 +1050,7 @@ bool OPUNetTransportLayer::SendUntilStatusUpdate(Packet *packet, int untilStatus
 		Sleep(repeatDelay);
 
 		// Pump the message receive processing
+		Packet dummyPacket;
 		while(Receive(&dummyPacket))
 			;

```

Comment:
Another not related change. Maybe move this semicolon to line 1054? Or maybe put in placeholder brackets. I think the semicolon is just signify that the body of the while loop doesn't exist, but maybe a more clear way to accomplish?

Reply:
Also, agreed that bStillWaiting isn't being used. Perhaps just remove for now?

Reply:
Placeholder brackets would be much much better. Moving the semicolon makes it really easy to miss what's happening here. I'd thought of making that change too, but figured it was a bit off topic at the time. Perhaps I'll throw in the change, since it's already been brought up.